build/lib/validate_artifacts.sh

```diff
@@ -35,6 +35,11 @@ if [ -n "$IMAGE_FORMAT" ]; then
   fi
 fi
 
+if ! stat $EXPECTED_FILES_PATH &> /dev/null; then
```

Comment:
this is risky, if we forget to add the file it will go unnoticed....

Reply:
Right. I'll try to skip it just for this case

Reply:
Yeah I fixed it, it was a silly thing